ghost/i18n/locales/zh/portal.json

```diff
@@ -132,22 +132,22 @@
     "Success! Your email is updated.": "成功！您的电子邮件已更新。",
     "Successfully unsubscribed": "成功取消订阅",
     "Thank you for subscribing. Before you start reading, below are a few other sites you may enjoy.": "感谢您的订阅。在开始阅读之前，以下是您可能会喜欢的一些其他网站。",
-    "Thank you for your support": "",
-    "Thank you for your support!": "",
+    "Thank you for your support": "感谢您的支持",
+    "Thank you for your support!": "感谢您的支持！",
     "Thanks for the feedback!": "感谢您的建议！",
     "That didn't go to plan": "似乎出错了",
     "The email address we have for you is {{memberEmail}} — if that's not correct, you can update it in your <button>account settings area</button>.": "您的电子邮件地址是 {{memberEmail}} - 如果该邮箱不正确，您可以在<button>帐户设置区域</button>中更新它。",
     "There was a problem submitting your feedback. Please try again a little later.": "提交您的反馈时遇到错误。请稍后重试。",
-    "There was an error processing your payment. Please try again.": "",
+    "There was an error processing your payment. Please try again.": "您的付款处理失败，请重试。",
     "This site is invite-only, contact the owner for access.": "此网站仅限邀请，联系网站所有者以获取访问",
-    "This site is not accepting payments at the moment.": "",
+    "This site is not accepting payments at the moment.": "本网站目前暂不接受付款。",
     "To complete signup, click the confirmation link in your inbox. If it doesn't arrive within 3 minutes, check your spam folder!": "要完成注册，请点击您收件箱中的确认链接。如果在3分钟内没有收到，请检查一下您的垃圾邮件文件夹！",
-    "To continue to stay up to date, subscribe to {{publication}} below.": "",
+    "To continue to stay up to date, subscribe to {{publication}} below.": "如需持续获取最新资讯，请在下方订阅{{publication}}",
     "Try free for {{amount}} days, then {{originalPrice}}.": "{{amount}}天免费试用，之后{{originalPrice}}。",
     "Unlock access to all newsletters by becoming a paid subscriber.": "成为付费订阅用户以解锁全部快报。",
     "Unsubscribe from all emails": "取消所有邮件订阅",
     "Unsubscribed": "取消订阅",
-    "Unsubscribed from all emails.": "",
+    "Unsubscribed from all emails.": "取消所有邮件订阅",
```

Comment:
Could you please recheck this one - it should describe the status of a user (the user is unsubscribeD), not offer the user an action to take.  (The action is above: "Unsubscribe from all emails")

Reply:
misled by the `Unsubscribed` translation, both fixed.